app/styles/atoms/input-container.less

```diff
@@ -45,10 +45,24 @@
     }
   }
 
-  &--errored {
```

Comment:
Changing `errored` to `error` 100% makes sense on a grammatical level, but are you sure we will not break a lot of tests outside the `oss-components` repo ?

Reply:
I have to admit that I didn't think about tests outside the repo
I will rollback on this

Reply:
I think Phil commented on how to apply this change while preventing tests from breaking in his comment here -> https://github.com/upfluence/oss-components/pull/453#discussion_r1860830065